requirements.txt

```diff
@@ -1,4 +1,4 @@
 bluepy==1.3.0
 pygatt==3.2.0
 btlewrap==0.0.9
-typing>=3,<4
+typing>=3,<4; python_version < 3.5
```

Comment:
```suggestion
```

Reply:
Is there still a point to this? Even python 3.5 is going end-of-life within a few months and Python 3.4 has been end-of-life for about a year already

Reply:
my suggestion was to remove it completely instead of limiting it, since based on `setup.py`, Python 3.4 is not supported.

Reply:
oops, I misread :)